Logger/Logger/ILogger.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    internal interface ILogger
+    {
+        public void LogInformation(string message);
+        public void LogWarning(string warningMessage);
```

Comment:
doar message, cuvantul warning deja exista in numele metodei

LogWarning(string message)